INSTRUCTION.md

```diff
@@ -0,0 +1,14 @@
+# ToDo App in Docker
+
+## Docker Hub Repository
+[Docker Hub Repository Link](https://hub.docker.com/repository/docker/beliar25/todoapp2.0/general)
+[Mysql](https://hub.docker.com/repository/docker/beliar25/mysql-test-db/general)
+
+## Build and Run Instructions
+
+### Build the Docker Image
+To build the Docker image, run the following command:
+```bash
+docker build . -f Dockerfile.mysql -t my-sql:1.0; docker run -d -p 3306:3306 --name my-mysql -v my-mysql-data:/var/lib/mysql my-sql:1.0; docker build . -f Dockerfile -t app:1.0; docker run -p 8080:8080 --name app app:1.0
```

Comment:
The command combines building and running multiple Docker images in one line. It's recommended to separate these commands for better readability and error handling. For example, use separate lines for building and running each image.